services/core-api/tests/auth/test_expected_auth.py

```diff
@@ -72,7 +72,7 @@
      (MineReportListResource, "post", [EDIT_REPORT, MINESPACE_PROPONENT]),
      (MineStatusXrefListResource, "get", [VIEW_ALL]),
      (MineTailingsStorageFacilityListResource, "get", [VIEW_ALL]),
-     (MineTailingsStorageFacilityListResource, "post", [MINE_EDIT]),
+     (MineTailingsStorageFacilityListResource, "post", [EDIT_TSF]),
```

Comment:
Should we also have another entry for the "put" given that it requires the same permissions?